docs/index.md

```diff
@@ -97,6 +97,28 @@ Now in order to access protected api urls you must include the `Authorization: B
 $ curl -H "Authorization: Bearer <your_token>" http://localhost:8000/protected-url/
 ```
 
+In addition to adding the Authorization: Bearer in your requests, make sure that you have decorated your views properly. For this, import the `JSONWebTokenAuthentication` authentication class from `rest_framework_jwt.authentication`
```

Comment:
You could also point out here that this is necessary if the `JSONWebTokenAuthentication` class wasn't added to the `DEFAULT_AUTHENTICATION_CLASSES`, or if JWT authentication is needed only on certain views.

Reply:
I just added a changelog file. Please review! 

(Also this is my first contribution on github. Thank you for being so helpful!)

Reply:
Congrats for your first contribution :slightly_smiling_face: 
As to the
> You could also point out here that this is necessary if the JSONWebTokenAuthentication class wasn't added to the DEFAULT_AUTHENTICATION_CLASSES, or if JWT authentication is needed only on certain views.

I don't see it anywhere. Did you forget to push that commit?

Reply:
I am really sorry, because of my unstable internet I have not been able to complete this. I will do this ASAP!

Reply:
I have added another commit that has addressed this missing part in the doc. Can you take a look?

Reply:
Great, thanks!